File: abstraccion/python/Electrodomesticos.py
class Lavadora:
    """Representa una Lavadora"""
    Variables de clase
    marca = ''
    modelo = ''
    nroSerie = 0
    capacidadMaximaKg = 0
    cargaActualKg = 0

    def __init__(self, marca, modelo, nroSerie, capacidad):
        # Variables de instancia
        self.marca = marca
        self.modelo = modelo
        self.nroSerie = nroSerie
        self.capacidadMaximaKg = capacidad

    def agregarRopa(self, pesoPrendaKg):
        if (pesoPrendaKg < self.capacidadMaximaKg):
            self.cargaActualKg += pesoPrendaKg

    def sacarRopa(self):
        self.cargaActualKg = 0

File: abstraccion/dart/lib/electrodomesticos.dart
class Lavadora {
  // Si la variable de instancia no es opcional, se debe inicializar.
  String marca = "";
  String modelo = "";
  int nroSerie = 0;
  double capacidadMaximaKg = 0.0;
  double cargaActual = 0.0;

  Lavadora(
      String marca, String modelo, int nroSerie, double capacidadMaximaKg) {
    this.marca = marca;
    this.modelo = modelo;
    this.nroSerie = nroSerie;
    this.capacidadMaximaKg = capacidadMaximaKg;
  }

  void agregarRopa(double pesoPrendaKg) {
    // ...
  }

  void sacarRopa() {
    // ...
  }
}
